dask_jobqueue/tests/test_pbs.py

```diff
@@ -7,10 +6,80 @@
 
 from dask_jobqueue import PBSCluster
 
-pytestmark = pytest.mark.env("pbs")
 
+#No default mark in order to run these simple unit tests all the time
```

Comment:
About comments: I would remove this kind of comments too. Maybe this behaviour of pytest can be surprising at first but for someone who has heard of pytest mark I think it makes sense. Also would you expect anyone that adds a test function to also put this comment before the new function?

Note: I would remove similar comments in test_slurm.py too.

Reply:
Okay I get it, I talk too much for myself in the code 😄